### PR TITLE
Convert project to React 

### PR DESCRIPTION
**Pull request by 3eak**
Contributes to: #33 

Signed-off-by: Stefan Cooper <stefan.cooper27@gmail.com>

Notes:

- Staged on https://koala-staging2.herokuapp.com/
- If you have any outstanding work that you want in, please open a PR ASAP
  - Should only be merged after #30

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from "@testing-library/react";
+import App from "./App";
+
+test("renders learn react link", () => {
```
Comment (ChrisF999) [inline src/App.test.js:5]: These test look to be generic for default template

File: package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "koala-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```
Comment (ChrisF999) [inline package.json:5]: some of theses are out of date such as 
Package        Current   Wanted  Latest  Location
node-sass       4.14.1   4.14.1   5.0.0  koala-react
react          16.14.0  16.14.0  17.0.1  koala-react
react-dom      16.14.0  16.14.0  17.0.1  koala-react
react-scripts    3.4.4    3.4.4   4.0.3  koala-react

File: src/reportWebVitals.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = (onPerfEntry) => {
```
Comment (ChrisF999) [inline src/reportWebVitals.js:1]: Is this file necessary, the call for it is commented out.

**Review by ChrisF999 — COMMENTED** [review]
This file appears to be empty.
src/styling/App.scss

File: src/App.test.js (hunk 1 above)
Comment (3eak) [reply src/App.test.js:5]: good spot - this was auto generated from `create-react-app`.

I will write a couple tests as examples later tonight for some of our files and remove these basic ones.

File: src/reportWebVitals.js (hunk 3 above)
Comment (3eak) [reply src/reportWebVitals.js:1]: i've left this in. this file can be used for measuring performance which may be useful later.

File: .travis.yml [hunk 4]
```diff
@@ -1,16 +1,13 @@
...
-    - JOBNAME=PYLINT
+      env:
+        - JOBNAME=ESLINT
 
-    install:
-        - pip install -r requirements.txt
-    scripts:
-        - pytest --pylint --ignore KoalaBot
+      scripts:
+        - npm run lint
```
Comment (ChrisF999) [inline .travis.yml:13]: surly we want to run the test suite now?

**Review by ChrisF999 — COMMENTED** [review]
All good one last little query

File: .travis.yml (hunk 4 above)
Comment (3eak) [reply .travis.yml:13]: ahhh good spot

**Review by ChrisF999 — APPROVED** [review]
Looks good to merge to me
